web/components/ResetButton.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <v-sheet class="transparent">
+    <v-btn color="error ma-2" dark @click="reset()"> Reset All Resources</v-btn>
+  </v-sheet>
+</template>
+
+<script lang="ts">
```

Comment:
Could you change this component to behave like delete button on ResourceBar? You can refer to the implementation of that.

I think the dialog is needed:
- not to reset mistakenly.
- to tell users that this reset will also reset the scheduler settings.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/web/components/ResourceBar/DeleteButton.vue